saleor/order/models.py

```diff
@@ -219,7 +214,7 @@ def __repr__(self):
     def __iter__(self):
         if self.id:
             return iter(self.lines.all())
-        return super().__iter__()
+        raise NotImplementedError()
```

Comment:
Why is this error raised here? I'm wondering what's the use case for that, do we need it?